qbicsoftware/sample-status-reporter
# Allow application to generate a log file based on a provided environment variable

**Introduced Changes**
Since the application employs `slf4j` based logging, it expects a `logback.xml` file instead of a `log4j2.xml` file. 
Additionally this PR introduces a dummy `last-updated.txt` resource file with an example timestamp that can be used for testing purposes. 

**How to test** 
Run the application with all environment variables set as introduced in the Readme. 
After a successful update check the now generated `sample-reporter-application.log` logfile in the specified LOG_PATH environment variable
